src/ddmd/e2ir.d

```diff
@@ -4847,7 +4847,7 @@ elem *toElem(Expression e, IRState *irs)
                     if (c1)
                     {
                         // Construct: (c1 || arrayBoundsError)
-                        auto ea = buildArrayBoundsError(irs, se.loc);
+                        auto ea = buildArrayBoundsError(irs, se.loc, el_copytree(elwr2), el_copytree(eupr2), el_copytree(elen));
```

Comment:
Yes, but I suggest waiting on fixing that, as I am opposed to this PR on performance grounds.